src/main/java/frc/robot/OI.java

```diff
@@ -48,11 +62,22 @@ public OI() {
     }
 
     private void createDriverCommands() {
-        drivetrain.setDefaultCommand(new DriveWithXbox(() -> driverXbox.getX(Hand.kLeft), driverXbox::getDeltaTriggers));
-        // TODO create driver commands here
+        driverDriveWithXbox = new DriveWithXbox(() -> driverXbox.getX(Hand.kLeft), driverXbox::getDeltaTriggers);
+        driverAutoShoot = new AutoShoot().withInterrupt(() -> Math.abs(driverXbox.getDeltaTriggers()) > 0.5);
+        driverCollectCell = new CollectCell();
+        driverCollectFromFeeder = new CollectFromFeeder()
+                .withInterrupt(() -> Math.abs(driverXbox.getDeltaTriggers()) > 0.5);
+        driverAutoClimb = new ClimbWithXbox(() -> 0, () -> robotConstants.climbConstants.kDefaultClimbPower);
+        driverRaiseHookWithXbox = new ClimbWithXbox(() -> driverXbox.getX(Hand.kRight), () -> 0);
```

Comment:
driverAutoClimb = new ClimbWithXbox(() -> 0, () -> robotConstants.climbConstants.kDefaultClimbPower);
        driverRaiseHookWithXbox = new ClimbWithXbox(() -> driverXbox.getX(Hand.kRight), () -> 0);

make one ClimbWithXbox command **Note** that the name o fthis command will change from climb testing branch.